docs/apidocs/template/pkg.tpl

```diff
@@ -41,9 +41,4 @@
     <hr/>
 {{ end }}
 
-<p><em>
-    Generated with <code>gen-crd-api-reference-docs</code>
-    {{ with .gitCommit }} on git commit <code>{{ . }}</code>{{ end }}.
-</em></p>
```

Comment:
I was thinking about changing these lines to:
```html
+<p><em>
    Generated with <code>gen-crd-api-reference-docs</code>
    on git commit <code>$Id$</code>.
</em></p>
```
and adding the following to `.gitattributes`:
```
docs/apidocs/template/pkg.tpl ident
```
This will generate the commit hash on the node at `git checkout`, but as we have different git SHAs at docs generation (before commit) and CI runs (after commit), will not solve the original "chicken and egg" problem.